### DISCUSSION TITLE
Game support?

### DISCUSSION THREAD

**Discussion by xDaswx** [IroniaTheMaster/Descrypt-global-metadata.dat#2] [General]
i can make this in every game?

**Comment by IroniaTheMaster** [comment]
> i can make this in every game?

No, not everyone has the same protection, unfortunately :(

**Comment by Void2166** [comment]
Hi @IroniaTheMaster 
Can u help me, pls, to descrypt my global-metada?
Cuz IL2CPP dumper is not working for my game, even that one that from magisk-zygisk that trying to catch that in game.
I want to open monobehaviour files in data unity 3d to create a translate for the game, but I can't do that, cuz of there is no way of dumping that shit.
